packages/dev/core/src/Animations/animatable.ts

```diff
@@ -866,7 +866,7 @@ Scene.prototype.stopAnimation = function (target: any, animationName?: string, t
  */
 Scene.prototype.stopAllAnimations = function (): void {
     if (this._activeAnimatables) {
-        for (let i = 0; i < this._activeAnimatables.length; i++) {
+        for (let i = this._activeAnimatables.length - 1; i >= 0; i--) {

```

Comment:
I think `this._activeAnimatables[i].stop` should be called like `this._activeAnimatables[i].stop(undefined, undefined, true)` instead, to avoid splicing the array (tiny perf gains).

Reply:
And then there is no need to change the iteration process